src/pages/MetricDetail.svelte

```diff
@@ -10,6 +10,34 @@
 
   const metricDataPromise = getMetricData(params.app, metricName);
 
+  function getGlamUrlTemplate(app) {
+    const map = {
+      fenix: {
+        product: "fenix",
+        app_id: "",
+      },
+      "firefox-android-beta": {
+        product: "fenix",
+        app_id: "beta",
+      },
+      "firefox-android-release": {
+        product: "fenix",
+        app_id: "release",
+      },
+    };
+    if (Object.keys(map).includes(app)) {
+      const p = map[app];
+      return function makeGlamUrl(metric) {
+        return `https://glam.telemetry.mozilla.org/${p.product}/probe/${metric}/explore?app_id=${p.app_id}`;
+      };
+    }
+
+    // The app isn't one GLAM supports so return nothing.
+    return null;
```

Comment:
`undefined` is a little more idiomatic here

```suggestion
    return undefined;
```